pages/home.html

```diff
@@ -167,9 +167,8 @@
 	<div class="background-image" style="background-image: url()"></div>
 	<div class="wrapper">
 		<div class="copy container">
-			<h1>The game engine you've been waiting&nbsp;for.</h1>
-			<p>The Godot Engine is a free, all-in-one, cross-platform game engine that makes it easy for you to create 2D and
-				3D games.</p>
+			<h1>Your free, open-source game engine.</h1>
```

Comment:
Could be solved along this idea:

```suggestion
			<h1><span style="white-space: nowrap;">Your free,</span> <span style="white-space: nowrap;">open-source</span> <span style="white-space: nowrap;">game engine.</span></h1>
```

Reply:
Thanks for the suggestion! I used instead non-breakable hyphen and `<br>`s.